### PR TITLE
Note: please note

### PR DESCRIPTION
**Pull request by Rick-Anderson**
too many notes

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@mairaw

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  [https://stage.docs.microsoft.com/en-us/dotnet/core/api/system.data.icolumnmappingcollection?branch=pr-en-us-1024](https://stage.docs.microsoft.com/en-us/dotnet/core/api/system.data.icolumnmappingcollection?branch=pr-en-us-1024)  
  [https://stage.docs.microsoft.com/en-us/dotnet/core/api/system.data.idataadapter?branch=pr-en-us-1024](https://stage.docs.microsoft.com/en-us/dotnet/core/api/system.data.idataadapter?branch=pr-en-us-1024)  
  [https://stage.docs.microsoft.com/en-us/dotnet/core/api/system.data.idataparameter?branch=pr-en-us-1024](https://stage.docs.microsoft.com/en-us/dotnet/core/api/system.data.idataparameter?branch=pr-en-us-1024)  
  Please see 3429 more published files of this docset in report.  

**Status**: Succeeded.  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C9%5C14%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201609142254554718-1024%5Cworkflow_report.html).

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ There are several ways to use DocFX, and most of them are covered in the [DocFX
 The following instructions use the [command-line based](https://dotnet.github.io/docfx/tutorial/docfx_getting_started.html#2-use-docfx-as-a-command-line-tool) version of the tool. 
 If you are comfortable with other ways listed on the link above, feel free to use those. 
 
-**Note:** please note that as of now, DocFX requires the .NET Framework on Windows or Mono for Linux or macOS. We'll be working towards porting it to .NET Core in the future. 
+**Note:** Currently DocFX requires the .NET Framework on Windows and Mono (for Linux or macOS). We hope to port it to .NET Core in the future.
```
Comment (svick) [inline CONTRIBUTING.md]: I think ".Net Framework on Mono" does not make sense, and that the previous wording of this part was better.

Comment (Rick-Anderson) [inline CONTRIBUTING.md]: The previous might be better but it's not good.  How about

DocFX requires the .NET Framework on Windows or Mono (for Linux or MacOS).

Comment (svick) [inline CONTRIBUTING.md]: Yeah, that sounds good to me.

Comment (mairaw) [inline CONTRIBUTING.md]: I like your suggestion @Rick-Anderson. After you change that, I can merge this.

**Pull request closed without merging**